katas/content/multi_qubit_gates/index.md

```diff
@@ -340,41 +336,41 @@ $$
 
 However, as $N$ gets larger, creating a full size matrix can be extremely unwieldy. To express the matrix without spelling out its elements, we can use the following trick:
 
-1. Apply the `SWAP` gate on the 1st and 2nd qubits.
-   This will bring the qubits on which the `CNOT` gate acts next to each other, without any extra qubits between them.
-2. Apply the `CNOT` on 2nd and 3rd qubits.
+1. Apply the $SWAP$ gate on the 1st and 2nd qubits.
+   This will bring the qubits on which the $CNOT$ gate acts next to each other, without any extra qubits between them.
+2. Apply the $CNOT$ on 2nd and 3rd qubits.
    Since now the gate acts on adjacent qubits, this can be represented as a tensor product of the gate we're applying and `I` gates.
-3. Apply the `SWAP` gate on the 1st and 2nd qubits again.
+3. Apply the $SWAP$ gate on the 1st and 2nd qubits again.
 
 These can be represented as applying the following gates on the 3 qubits.
 
-1. $\text{SWAP} \otimes I$
+1. $SWAP \otimes I$
 
 $$
 x_{000}|000\rangle + x_{001}|001\rangle + x_{100}|010\rangle + x_{101}|011\rangle +
 x_{010}|100\rangle + x_{011}|101\rangle + x_{110}|110\rangle + x_{111}|111\rangle
 $$
 
-2. $I \otimes \text{CNOT}$
+1. $I \otimes CNOT$
 
 $$
 x_{000}|000\rangle + x_{001}|001\rangle + x_{101}|010\rangle + x_{100}|011\rangle +
 x_{010}|100\\rangle + x_{011}|101\rangle + x_{111}|110\rangle + x_{110}|111\rangle
 $$
 
-3. $\text{SWAP} \otimes I$
+1. $SWAP \otimes I$
```

Comment:
```suggestion
3. $SWAP \otimes I$
```

Reply:
Maybe the IDE changed the numbers automatically?

Reply:
Something automatic definitely happened here, I know I didn't do that by hand... Good catch!